src/sentry/tasks/merge.py

```diff
@@ -12,7 +14,7 @@
 delete_logger = logging.getLogger("sentry.deletions.async")
 
 
-EXTRA_MERGE_MODELS = []
+EXTRA_MERGE_MODELS: list[Any] = []
```

Comment:
this shouldn't be `list[Any]` -- this should use the actual type here -- I think it's maybe `list[type[Model]]` ?